special-use/special-use-suds-mapping.json

```diff
@@ -6,97 +6,97 @@
       "applicant-info": {
         "type": "object",
         "properties": {
-      		"firstName": { "type": "string", "suds-application": "Contacts", "suds-field": "First Name"},
-      		"lastName": {	"type": "string", "suds-application": "Contact", "suds-field": "Last Name" },
+      		"firstName": { "type": "string", "basic-api-field" : "FIRST_NAME"},
+      		"lastName": {	"type": "string", "basic-api-field" : "LAST_NAME" },
```

Comment:
I'm not sure how to display, but this would not be a required field if an existing `CONT_ID` is supplied.